ext/curl/interface.c

```diff
@@ -750,6 +750,8 @@ PHP_MINIT_FUNCTION(curl)
 	REGISTER_CURL_CONSTANT(CURLE_ABORTED_BY_CALLBACK);
 	REGISTER_CURL_CONSTANT(CURLE_BAD_CALLING_ORDER);
 	REGISTER_CURL_CONSTANT(CURLE_BAD_CONTENT_ENCODING);
+	REGISTER_CURL_CONSTANT(CURLE_CONV_FAILED);
+	REGISTER_CURL_CONSTANT(CURLE_CONV_REQD);
```

Comment:
Hm, some of these constants (such as these two) are not relevant for end users, because the character conversion functionality is not exported by PHP (and won't be). I'm wondering whether or not we should still define them.

Reply:
I don't know about the use case for these constants, but IMO we should blacklist them given your explanation.
If the extension isn't really capable to get this kind of errors, the only use case I can imagine is to compare error codes returned by userland code, maybe a library using the system's cURL library directly via `passthru()` or similar.

Reply:
If the decision is to not include these constants, they must be added to `IGNORED_CONSTANTS`: https://github.com/php/php-src/blob/906331a2007ffd8a820b981a4919d1454727a3ea/ext/curl/sync-constants.php#L16-L18

Reply:
I also think they shouldn't be exposed.